solutions/USACO jan21-even-more-odd-photos-bronze.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: usaco-1084
+source: USACO Bronze 2021 January
+title: Even More Odd Photos
+author: Sathvik Chundru
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob2_bronze_jan21.html)
+
+
+</LanguageSection>
+<JavaSection />
+
+```java
+import java.util.*;
+import java.io.*;
+public class Main {
+    public static void main(String[] args) {
+      Scanner sc = new Scanner(System.in);
+        int O=0;
+        int E = 0;
+        int x;
+        int n = sc.nextInt();
+        for (int i=0; i<n; i++) {
+          x = sc.nextInt();
+          if (x % 2 == 0) {
+            E++;
+          } 
+          else{
+            O++;
+          }
+        }
+      while (O > E) { 
+        O=O-2; E++; 
+      }
+      if (E > O+1) {
+        E = O+1;
+      }
+      System.out.println(E + O);
+
+  }
+       
+ 
+}
+```
+
+</JavaSection>
```

Comment:
```suggestion
</JavaSection>

```

Reply:
technically not necessary but whitespace is good :p

Reply:
sus